docs/hugo/content/contributing/generator-overview.md

```diff
@@ -0,0 +1,49 @@
+# Code Generator Overview
+
+Core to Azure Service Operator (ASO) v2 is our code generator. This consumes ARM JSON Schema and Swagger specifications and generates the code for each desired resource.
+
+## Code Structure
+
+The key packages used to structure the code of the generator are as follows:
```

Comment:
This is very helpful to get a clear idea of where to look at for the code-gen packages